src/Checkbox/styles/index.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+export const parsePxValue = (pxString: string) => parseFloat(pxString);
+export const parseStringToFloat = (stringValue: string) =>
+  parseFloat(stringValue);
+
```

Comment:
Basically I currently treat all style related stuff more like 'work in progress' which will be probably replaced by some global solution. 
Thats why I simply wanted to have all style parsing related stuff in one place. So it will be easier to update/replace it in a future.
And I'm also still not really sure about using just `parseFloat` for design-tokens values parsing, so thats another reason why I wanted to have it wrapped, so it can be easily switched when we decide to use some different approach.